docs/fundamentals/application-insights/sample/src/AppInsightsDemo/Startup.cs

```diff
@@ -0,0 +1,131 @@
+﻿using Microsoft.AspNet.Authentication.Facebook;
+using Microsoft.AspNet.Authentication.MicrosoftAccount;
+using Microsoft.AspNet.Builder;
+using Microsoft.Framework.DependencyInjection;
+using Microsoft.AspNet.Diagnostics;
+using Microsoft.AspNet.Diagnostics.Entity;
+using Microsoft.AspNet.Hosting;
+using Microsoft.AspNet.Identity.EntityFramework;
+using Microsoft.Data.Entity;
+using Microsoft.Framework.ConfigurationModel;
+using Microsoft.Framework.Logging;
+using AppInsightsDemo.Models;
+using Microsoft.ApplicationInsights.AspNet;
+
+namespace AppInsightsDemo
+{
+    public class Startup
+    {
+        public Startup(IHostingEnvironment env)
+        {
+            // Setup configuration sources.
+            var configuration = new Configuration()
+                .AddJsonFile("config.json")
+                .AddJsonFile($"config.{env.EnvironmentName}.json", optional: true);
+
+            if (env.IsEnvironment("Development"))
+            {
+                // This reads the configuration keys from the secret store.
+                // For more details on using the user secret store see http://go.microsoft.com/fwlink/?LinkID=532709
+                configuration.AddUserSecrets();
+                configuration.AddApplicationInsightsSettings(developerMode: true);
```

Comment:
It will be nice to add a comment above this line saying:

"//Setting development mode will expedite your telemetry through the pipeline so that you can see results immediately: http://azure.microsoft.com/en-us/documentation/articles/app-insights-api-custom-events-metrics/#debug. Consider adding this link and test in the documentation itself